examples/08-rust-kernel.rs

```diff
@@ -34,6 +43,8 @@ fn main() -> Result<(), DriverError> {
     println!("b_host {b_host:?}");
     
     println!("a_host_2 {a_host_2:?}");
+
+    println!("cleaned successfully? {:?}", rust_ptx.clean());
```

Comment:
PtxCrate also cleans up after itself if you ask it to. Potentially this could be added to the destructor but for now it's opt-in in case users want to keep their residual ptx.